jest.config.js

```diff
@@ -4,7 +4,7 @@ module.exports = {
   },
   moduleNameMapper: {
     '.+\\.(css|styl|less|sass|scss)$': 'identity-obj-proxy',
-    '.+\\.(jpg|jpeg|png|gif|eot|otf|webp|svg|ttf|woff|woff2|mp4|webm|wav|mp3|m4a|aac|oga)$': '<rootDir>/__mocks__/file-mock.js',
+    '.+\\.(jpg|jpeg|png|gif|eot|otf|webp|svg|ttf|woff|woff2|mp4|webm|wav|mp3|m4a|aac|oga)$': '<rootDir>/fileTransformer.js',
```

Comment:
EdxLogo as svg fails tests when trying to using file-mock.js (I'm not entirely sure if this file even exists). Based on documentation, I added fileTransformer file to mock the svg file. 

https://jestjs.io/docs/en/webpack